appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -242,7 +241,7 @@ public class Appcues internal constructor(koinScope: Scope) {
      * @param frameId unique string set in builder that is used to identify this embed view with qualified experiences
      * @param view frame used to inflate the embedded experience
      */
-    public fun registerEmbed(frameId: String, view: ViewGroup) {
+    public fun registerEmbed(frameId: String, view: AppcuesEmbedView) {
```

Comment:
we've been referring to this as the `Frame` in documents, and the `frameID` in the json. iOS calls it the `AppcuesFrameView` and has a register frame function - I think would be ideal to stick to that nomenclature.

Reply:
can surely change from AppcuesEmbedView to AppcuesFrameView